gpuz.txt

```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = TechPowerUp GPU-Z 
```

Comment:
Please strip the EOL whitespace at the end of line 2.
Afterwards I will merge the PR.

Reply:
done